plugins/oracledb_metrics.yaml

```
@@ -1,4 +1,4 @@
-version: 0.0.1
+version: 0.1.0
```

```suggestion
version: 0.0.2
```

This doesn't seem like a minor version update since no new functionality was added, seems like a fix :smile

---

@schmikei I told @antonblock to bump the minor version. This is the current way we've been versioning these. The very first version is `0.0.1` then the first change after that is `0.1.0`. I realize this is clear as mud and we should probably do a better more defined way of versioning this in the future.